README.rst

```diff
@@ -96,13 +96,15 @@ Quick Example
     from nextcord.ext import commands
 
 
-    bot = commands.Bot(command_prefix='$')
+    bot = commands.Bot()
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
 
-    bot.run('token')
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!", ephemeral=True)
+
+    bot.run("token")
+
```

Comment:
I know this is really picky but its annoying me, could you remove this extra newline?

Reply:
Done!

Reply:
i meant the other way round, leave the newline before `bot.run` but remove the added newline after it.

Reply:
Check now.